packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -133,6 +134,16 @@ export const BookingFields = ({
           <FormBuilderField className="mb-4" field={{ ...field, hidden }} readOnly={readOnly} key={index} />
         );
       })}
+      {/* New Waitlist Checkbox */}
+      <div className="mb-4">
+        <label>
+          <input
+            type="checkbox"
+            {...register("waitlistOptIn")} // Integrate checkbox with react-hook-form
+          />
+          Sign me up for a waitlist for earlier times if available
```

Comment:
This text should be added to the `description` of the `CheckboxField`, also we are missing translations here. You only have to add a new translation key for english in`locales/en/common.json`